### ISSUE TITLE
Missing confirmation of Alt Text selection

### ISSUE THREAD

**Issue by zqian** [tl-its-umich-edu/canvas-app-explorer#652]
Based on the [Accessibility Review Report](https://docs.google.com/spreadsheets/d/1vt4uJuSq3itNu8Ouq6KI-4KOdOlOi_T9PoqOVy0e3T0):

Severity: 4- Critical 

Test: Review Alt Text 

Issue notes: "Set X alt text labels as" - there is no confirmation (visual or for screen reader users) of what option the user has selected (it doesn't stay visible in the box after selection).

Category: Semantics and Structure

Criterion: [4.1.1 - Name, Role, Value](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1#name-role-value)

Impact: People who use assistive technology may not be able to use content or controls.

Most affects people who are/have: Blind, Low vision, Mobility disability, Neurodivergent

**Comment by jonespm** [comment]
So in looking at this I did come up with a potential fix
* Improved the bulk “Set X alt text labels as” control so users can see their selection, get confirmation, and receive better accessibility feedback.
<img width="1223" height="109" alt="Image" src="https://github.com/user-attachments/assets/79c0068a-c8d7-4f6b-88a2-38c8faabeb19" />

However I noticed some other potential issue around this combo-box's behavior that we may want to improve.

* If there are manual edits, the bulk action overrides this. It seems like bulk action should only apply to items that the user hasn't changed. 
* Doing this would need to make this count dynamic so it reflects items still eligible for bulk changes
* There's no way to "reset" a card or the changes made. It seems like there should be a way to undo specific items and the bulk action without going back.

I believe I have implemented all of these 3 suggestions here but still testing and will talk about it for feedback.

**Comment by jonespm** [comment]
Decisions here: 

Remove individual card clear action
Update bulk actions text to apply to all items
